zerver/tests/test_auth_backends.py

```diff
@@ -1185,7 +1185,8 @@ def test_wrong_password(self):
                                        password="wrong"))
         self.assert_json_error(result, "Your username or password is incorrect.", 403)
 
-    @override_settings(AUTHENTICATION_BACKENDS=('zproject.backends.GoogleMobileOauth2Backend',))
+    @override_settings(AUTHENTICATION_BACKENDS=('zproject.backends.GoogleMobileOauth2Backend',),
+                       SEND_LOGIN_EMAILS=True)
```

Comment:
We should add some verification in the test that the email was sent, using the outbox.